maintainers/maintainer-list.nix

```diff
@@ -5006,6 +5006,15 @@
       fingerprint = "9484 44FC E03B 05BA 5AB0  591E C37B 1C1D 44C7 86EE";
     }];
   };
+  dustinblackman = {
```

Comment:
Please, add yourself to maintainers list as separate commit.

Reply:
The commit should be called `maintainers: add dustinblackman`.